src/templates/UserList/UserListToolbar.tsx

```diff
@@ -36,7 +37,7 @@ export function UserListToolbar(props: UserListToolbarProps) {
       {numSelected > 0 ? (
         <>
           {selectedRows.every((row) => {
-            return row.userType === 'Pending';
+            return row.roles?.includes({ id: '0', name: 'admin' });
```

Comment:
I think this is incorrect

roles should be an array of roles(strings) and now you are trying to fin and object + one more thing. Even if you try to find an object, you can not compare values like that (vale vs refernce ;))

```javascript
const a = { age: 12 };
const b = { age: 12 };

a === b; // always return false
```

Reply:
According to the generated types, roles are an array of objects.
![image](https://user-images.githubusercontent.com/72614768/235323227-3cfce13d-9057-43e6-bb49-29fd0391e317.png)

Reply:
fixed in [ac83857](https://github.com/dorm-social-room-manager/dsrm-frontend/pull/38/commits/ac838579c3e681b31ed846e2ad54f55fc2a734e6)